lca-cli/ops/ops.go

```diff
@@ -115,7 +118,29 @@ func (o *ops) ForceExpireSeedCrypto(recertContainerImage, authFile string, hasKu
 	if err := recert.CreateRecertConfigFileForSeedCreation(recertConfigFile, hasKubeAdminPassword); err != nil {
 		return fmt.Errorf("failed to create %s file: %w", recertConfigFile, err)
 	}
-	if err := o.RecertFullFlow(recertContainerImage, authFile, recertConfigFile, nil, nil); err != nil {
+
+	// Run post recert operation to defragment etcd after recert tool is run
+	// this should allow for a more efficient etcd work after installation
+	// should not fail seed creation for now
```

Comment:
You will also need to run a compaction to have the defrag be useful. This is an example of using the latest revision. You could perform this during seed and it would improve I/O latency. Tradeoff being defrag is blocking but if you did it periodically during seed the duration would be minimal.

```
	resp, _ := client.KV.Get(ctx, "key")
	latestRevision := resp.Header.Revision

	_, err = client.Compact(ctx, latestRevision)
```

Reply:
Note: Compaction is not required in OCP/K8s because it is done periodically every 5m by the ApiServer.

Reply:
do you think we need to run it here? I assume it will run 5 minutes after we will start the cluster, no? 
I mean that if in seed we run it every 5 minutes it is compacted already so we just need to defrag it. From what i see in doc it say compact and then defrag. IMO don't see a reason to add compact here in this case but i really can miss something here

Reply:
If the goal is to improve seed timing, then consider it. Post seed apiserver and etcd-operator manage this as you noted.  Compacted db will read/write faster in general so this is the problem we are solving for.

Reply:
Talking with @tsorya I misunderstood seed step as not having an apiserver. In which case this would be useful. If the cluster is otherwise running then compaction would not be necessary step.